From 28cb73441ec6f6e230e69070e1110da6fbbe41a9 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Fri, 19 Oct 2018 12:19:02 +0200
Subject: [PATCH] Merge android-9.0.0_r12

commit 6616587a42ba2e367cb854b037ce3a06fee95522
Merge: 8da1fbb 94f7e57
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Fri Jul 20 23:43:58 2018 +0000

    Snap for 4905103 from 94f7e57d68b07f9fbfc7d15ad06c5467a4bc0afc to pi-dr1-release

    Change-Id: I3e0e2788b7e23f90f8b9fcfb11ec50bec3b2eeff

commit 94f7e57d68b07f9fbfc7d15ad06c5467a4bc0afc
Author: Bill Yi <byi@google.com>
Date:   Fri Jul 20 12:00:11 2018 -0700

    Import translations. DO NOT MERGE

    Auto-generated-cl: translation import

    Bug: 64712476
    Change-Id: I4ded0f91d9aa98ace6a47aae11f97d5da5555e94

Change-Id: Ib92c808b86f6648722c12ce92c73dcf60d6950bf
---
 res/values-sw/strings.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/res/values-sw/strings.xml b/res/values-sw/strings.xml
index 186a561..b6ee5f5 100644
--- a/res/values-sw/strings.xml
+++ b/res/values-sw/strings.xml
@@ -17,7 +17,7 @@
 
 <resources xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="app_name" msgid="3551052199033657984">"Huduma Chaguo msingi ya Kuchapisha"</string>
+    <string name="app_name" msgid="3551052199033657984">"Huduma Chaguomsingi ya Kuchapisha"</string>
     <string name="printer_busy" msgid="8604311528104955859">"Inatumika"</string>
     <string name="printer_out_of_paper" msgid="4882186432807703877">"Karatasi zimeisha"</string>
     <string name="printer_out_of_ink" msgid="7361897651097675464">"Wino umeisha"</string>
-- 
2.17.1

